modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -169,11 +169,13 @@ public static real_t Lerp(real_t from, real_t to, real_t weight)
             return from + (to - from) * weight;
         }
 
-        public static real_t LerpAngle(real_t from, real_t to, real_t weight)
-        {
-            real_t difference = (to - from) % Mathf.Tau;
-            real_t distance = ((2 * difference) % Mathf.Tau) - difference;
-            return from + distance * weight;
+        public static real_t LerpAngle(real_t from, real_t to, real_t weight) {
+            return from + Mathf::AngleDiff(to, from) * weight;
+        }
+
+        public static real_t AngleDiff(real_t minuend, real_t subtrahend) {
```

Comment:
In C# we use the Allman brace style, the opening brace needs to go on the next line.

I can't tell what `minuend` and `subtrahend` are supposed to be. Why not call them `from` and `to`?

Reply:
`minuend` and `subtrahend` are the terms used to for subtraction operands, so I decided to use them. Formating the braces.

Reply:
Aside from those being very uncommon words, it's missing the point of argument names. The user doesn't need to know the implementation details (that the numbers are used in subtraction) but rather what their purpose is (that the arguments are interpreted as from and to angles so that the difference can be found).